config/matomoWalletsEvents.ts

```diff
@@ -0,0 +1,330 @@
+    'Lido_Frontend_Template',
+    'Click imToken wallet',
+    'eth_widget_click_imtoken',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectLedger]: [
+    'Lido_Frontend_Template',
+    'Connect Ledger wallet',
+    'eth_widget_connect_ledger',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickLedger]: [
+    'Lido_Frontend_Template',
+    'Click Ledger wallet',
+    'eth_widget_click_ledger',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectMathWallet]: [
+    'Lido_Frontend_Template',
+    'Connect MathWallet wallet',
+    'eth_widget_connect_mathwallet',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickMathWallet]: [
+    'Lido_Frontend_Template',
+    'Click MathWallet wallet',
+    'eth_widget_click_mathwallet',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectMetamask]: [
+    'Lido_Frontend_Template',
+    'Connect Metamask wallet',
+    'eth_widget_connect_metamask',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickMetamask]: [
+    'Lido_Frontend_Template',
+    'Click Metamask wallet',
+    'eth_widget_click_metamask',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectOperaWallet]: [
+    'Lido_Frontend_Template',
+    'Connect Opera wallet',
+    'eth_widget_connect_opera',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickOperaWallet]: [
+    'Lido_Frontend_Template',
+    'Click Opera wallet',
+    'eth_widget_click_opera',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectTally]: [
+    'Lido_Frontend_Template',
+    'Connect Tally wallet',
+    'eth_widget_connect_tally',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickTally]: [
+    'Lido_Frontend_Template',
+    'Click Tally wallet',
+    'eth_widget_click_tally',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectTrust]: [
+    'Lido_Frontend_Template',
+    'Connect Trust wallet',
+    'eth_widget_connect_trust',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickTrust]: [
+    'Lido_Frontend_Template',
+    'Click Trust wallet',
+    'eth_widget_click_trust',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectWC]: [
+    'Lido_Frontend_Template',
+    'Connect WalletConnect wallet',
+    'eth_widget_connect_walletconnect',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickWC]: [
+    'Lido_Frontend_Template',
+    'Click WalletConnect wallet',
+    'eth_widget_click_walletconnect',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectXdefi]: [
+    'Lido_Frontend_Template',
+    'Connect XDEFI wallet',
+    'eth_widget_connect_xdefi',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickXdefi]: [
+    'Lido_Frontend_Template',
+    'Click XDEFI wallet',
+    'eth_widget_click_xdefi',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onConnectZenGo]: [
+    'Lido_Frontend_Template',
+    'Connect ZenGo wallet',
+    'eth_widget_connect_zengo',
+  ],
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickZenGo]: [
+    'Lido_Frontend_Template',
+    'Click ZenGo wallet',
+    'eth_widget_click_zengo',
+  ],
+};
+
+export const walletsMetrics: WalletsMetrics = {
+  events: {
+    click: {
+      handlers: {
+        onClickAmbire: () => {
```

Comment:
better to introduce some local fn like this:
```typescript
const bindEvent = (event) => () => trackEvent(event)
```

Reply:
Looks like overheads.

I can't see the difference between this one:
```
const bindEvent = (event) => () => trackEvent(...event);

onClickAmbire: () => {
    bindEvent(MATOMO_WALLETS_EVENTS.onClickAmbire);
},
```

and second one:
```
onClickAmbire: () => {
    trackEvent(...MATOMO_WALLETS_EVENTS.onClickAmbire);
},
```

Or give more context, pls